packages/devtools_app/lib/src/screens/memory/panes/profile/profile_view.dart

```diff
@@ -204,6 +205,209 @@ class _FieldSizeColumn extends ColumnData<ProfileRecord> {
   bool get numeric => true;
 }
 
+abstract class _HeapStatsColumn extends ColumnData<AdaptedProfile> {
+  _HeapStatsColumn(
+    super.title, {
+    required this.generation,
+    required super.fixedWidthPx,
+    super.titleTooltip,
+    super.alignment,
+  });
+
+  final _HeapGeneration generation;
+
+  GCStats getGCStats(AdaptedProfile profile) {
+    switch (generation) {
+      case _HeapGeneration.newSpace:
+        return profile.newSpaceGCStats;
+      case _HeapGeneration.oldSpace:
+        return profile.oldSpaceGCStats;
+      case _HeapGeneration.total:
+        return profile.totalGCStats;
+    }
+  }
+}
+
+class _HeapNameColumn extends ColumnData<AdaptedProfile> {
+  _HeapNameColumn()
+      : super(
+          '',
+          fixedWidthPx: scaleByFontFactor(200),
+        );
+
+  @override
+  String? getValue(AdaptedProfile dataObject) {
+    return 'GC Statistics';
+  }
+}
+
+class _HeapUsageColumn extends _HeapStatsColumn {
```

Comment:
same with all of these _GCUsageColumn, _GCCapacityColumn, _GCLatencyColumn to name them more appropriately with the column titles

Reply:
hmm actually maybe I'm wrong on _HeapUsageColumn and _HeapCapacityColumn I don't think I understood the context. However, I still think _GCLatencyColumn is a more fitting name